frontend/src/metabase/models/containers/ModelDetailPage/ModelDetailPage.unit.spec.tsx

```diff
@@ -83,7 +125,9 @@ async function setup({ model, collections = [], usedBy = [] }: SetupOpts) {
   const slug = `${card.id}-model-name`;
 
   if (database) {
-    setupDatabasesEndpoints(scope, [getDatabaseObject(database)]);
+    setupDatabasesEndpoints(scope, [
+      getDatabaseObject(database, { hasActionsEnabled }),
```

Comment:
Could you refactor this test suite to only use mocked data and not entities from `metabase-lib`?